File: codeforces/703A Mishka and Game.py
t = int(input())
c = 0
m = 0
while t > 0:
	x = input().split()
	a = int(x[0])
	b = int(x[1])
	if a > b:
		m += 1
	elif b > a:
		c += 1
	t += -1
if m > c:
	print('Mishka')
elif c > m:
	print('Chris')
else:
	print('Friendship is magic!^^')

File: codeforces/703A Mishka and Game.cpp
#include <iostream>
using namespace std;

int main() {

	int t, c = 0, m = 0;
	cin >> t;
	while (t--) {
		int a, b;
		cin >> a >> b;
		if (a > b)
			m++;
		else if (b > a)
			c++;
	}
	if (m > c)
		cout << "Mishka" << endl;
	else if (c > m)
		cout << "Chris" << endl;
	else
		cout << "Friendship is magic!^^";

}